frida_mode/GNUmakefile

```diff
@@ -10,11 +10,19 @@ OBJS:=$(foreach src,$(SOURCES),$(OBJ_DIR)$(notdir $(patsubst %.c, %.o, $(src))))
 CFLAGS+=-fPIC \
 		-D_GNU_SOURCE \
 		-D_FORTIFY_SOURCE=2 \
-		-Wno-pointer-arith \
+		-Werror \
+		-Wall \
+		-Wextra \
+		-Wpointer-arith \
```

Comment:
please remove -Werror -Wall and -Wextra
this is great if you are developing, but it should not be in the branch.
also rather set -Wno-pointer-arith

Reply:
Ok no worries. Are you ok if I keep them e.g. if you build with ‘DEV=1 make’ or something they are enabled. Otherwise they aren’t?

Reply:
Or maybe a separate (non-default) make target which enables the options?

Reply:
as you want "make debug" or "make DEBUG=1" or dev/DEV ... as you like